doc/ansible/dev/advanced_options.md

```diff
@@ -50,3 +50,28 @@ You may want to manage what your operator watches and the owner references. This
 ```
 ENTRYPOINT ["/usr/local/bin/entrypoint", "--inject-owner-ref=false"]
 ```
+
+## Max Workers
+
```

Comment:
I think it still missing when should a user use it? What it is the purpose?  WDYT?